CHANGELOG_iOS.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
```

Comment:
We usually use `Updated` when mentioning a feature that was overhauled re: new UI. Can probably just do something simple as per the following:

```
  - Updated Playlist UI. ([#40911](https://github.com/brave/brave-browser/issues/40911))ues/36801))
```